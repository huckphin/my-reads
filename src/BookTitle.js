import React, { Component } from 'react';
import PropTypes from 'prop-types';

class BookTitle extends Component {
    static propTypes = {
        title: PropTypes.string.isRequired
    }

    render () {
        const { title } = this.props;

        return (
            <div className="book-title">{{ title }}</div>
        );
    }
}

export default BookTitle;