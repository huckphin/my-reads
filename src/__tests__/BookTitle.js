import React from 'react';
import BookTitle from '../BookTitle'
import renderer from 'react-test-renderer'

test('render a BookTitle', () => {
    const component = renderer.create(
        <BookTitle title="Fight Club" />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});