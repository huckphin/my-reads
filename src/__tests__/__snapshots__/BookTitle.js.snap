// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a BookTitle 1`] = `
<div
  className="book-title"
>
  Fight Club
</div>
`;
